docs/setup.md

```diff
@@ -1,18 +1,23 @@
 
 # Setting up ClintLR
 
-ClintLR is a desktop Java application that includes LIRICAL analysis. Therefore, ClintLR requires LIRICAL to be installed.
-Instructions for installing LIRICAL can be found [here](https://thejacksonlaboratory.github.io/LIRICAL/stable/setup.html#rstsetup).
-Similar to LIRICAL, ClintLR also requires Exomiser to be installed as a library before it can be compiled and built.
-
-
 ## Prerequisites
 
 ClintLR was written with Java version 17 but will compile under Java 17 or higher. If you want to
 build ClintLR from source, then the build process described below requires
 [Git](https://git-scm.com/book/en/v2) and
 [Java Development Kit 17](https://www.oracle.com/java/technologies/downloads/) or higher.
+ClintLR also requires Exomiser to be installed as a library before it can be compiled and built.
+
 
+## Installing ClintLR
+
+Executable JAR files for Linux, Mac, and Windows platforms can be downloaded from the
+[releases](https://github.com/TheJacksonLaboratory/ClintLR/releases) page. 
+
+**Note:** In order to run ClintLR using the JAR for Mac OS with ARM architecture (e.g. Apple M1 chip), LIRICAL needs to
+be installed separately. Instructions for installing LIRICAL as a library can be found
+[here](https://thejacksonlaboratory.github.io/LIRICAL/stable/setup.html#building-to-use-as-a-library).
 
 
 ## Building from sources
```

Comment:
This section seems unfinished. You must describe how to install LIRICAL first, referring to the docs section I added today.

After that, you must instruct the users to clone ClintLR, do `./mvnw package`, and describe where they will find the GUI JAR file.